### ISSUE TITLE
Tabs overflowing on tab bar

### ISSUE THREAD

**Issue by dbfreem** [minbrowser/min#694]
I am not sure if this is a bug or I am doing something wrong. When I open more tabs than can fit on the tab bar, I can't drag to get to the hidden tabs. See the screenshot below.

![screenshot from 2019-01-06 21-21-12](https://user-images.githubusercontent.com/9778569/50745632-514c6200-11f9-11e9-9c46-1aaa63ccf35d.png)

**Comment by code-hunger** [comment]
You'll see the tabs better on the task overlay anyway (Ctrl + E). And also you'll be able to directly jump to any of them without the need for scrolling.

**Comment by PalmerAL** [comment]
The task overlay probably is better if you have lots of tabs, but it's still nice to have a way to scroll without opening it.

Dragging the scrollbar works fine for me on Windows and Linux (it's broken on macOS, but that's a separate issue), but it's really short, which makes it difficult to select with the mouse. We could probably make it taller, and maybe reduce the opacity a bit so it doesn't stand out as much. If you position the top of your mouse cursor right over the scrollbar, can you drag it?

**Comment by dbfreem** [comment]
@PalmerAL I tried several times to drag the scroll bar and was finally able to get it to scroll. Probably, one out of 10 times I can get it to scroll. I am running Ubuntu. I guess one option is to make it a little bit bigger in order to be able to drag it appropriatly but that would also have the side effect of being too visible.

**Comment by PalmerAL** [comment]
I think we could make it bigger if we reduced the contrast of the scrollbar:

<img width="877" alt="screen shot 2019-01-07 at 8 51 46 pm" src="https://user-images.githubusercontent.com/10314059/50807325-8622ec80-12bf-11e9-86cf-43feb58cef3a.png">
<img width="877" alt="screen shot 2019-01-07 at 9 01 27 pm" src="https://user-images.githubusercontent.com/10314059/50807324-8622ec80-12bf-11e9-9a80-9d1997796d4e.png">

(we could also reduce the contrast even more than that, and then increase it when you hover over the tab strip).

Another option would be to increase the size of the scrollbar on hover, although I think this would be distracting if you're just trying to click on a tab.

**Comment by Thalos12** [comment]
Instead of dragging the scrollbar, I found that I can use Shift+scroll to move the tabs if they are overflowing, but this works only when scrolling down (which corresponds to showing the hidden tabs on the right). Scrolling up on the other hand (even holding shift) open the task overlay.
I will open a new issue if it is needed.

**Comment by PalmerAL** [comment]
@Thalos12 Scrolling up or down is supposed to open the task overlay (or close the current tab); we probably shouldn't do that if you're holding the shift key though.

**Comment by Thalos12** [comment]
@PalmerAL I know about the scrolling feature (actually I disabled closing tabs with scroll). I will open a separate issue for the shift+scroll behaviour.

**Comment by PalmerAL** [comment]
@dbfreem I've updated the scrollbar styling in ecfb158f2d8c7981e5d76220e42c28e74c75ea56 to match the screenshot I posted above, could you try that out and see if it helps?

**Comment by dbfreem** [comment]
@PalmerAL I built the most recent master branch and I am still seeing and issue. It seems like there might be something wrong with the events firing on the scroll bar for the tabs. In the screenshot below `context` keeps returning null and the `grab` function just returns without dragging the scroll bar. Again I might be missing something but it still seems like when I click to drag the scroll bar over 50% of the time it doesn't drag. 

![image](https://user-images.githubusercontent.com/9778569/51363815-b3358300-1aa7-11e9-8d3d-87be8a53d3ba.png)

**Comment by dbfreem** [comment]
@PalmerAL I tried out ecfb158. I think something else is going on here. Take a look at the two screenshots below. The first one when I click directly in the middle of the scrollbar it doesn't scroll. In the second one when I click barely above the scroll bar it scrolls everytime. 

![screenshot from 2019-01-24 21-40-47](https://user-images.githubusercontent.com/9778569/51721709-e71d2500-2020-11e9-96f2-701b6279f1ac.png)


![screenshot from 2019-01-24 21-41-17](https://user-images.githubusercontent.com/9778569/51721713-e8e6e880-2020-11e9-8fd1-3bf8efa4a548.png)

**Comment by PalmerAL** [comment]
@dbfreem You're right, it looks like the BrowserView is preventing the scrollbar from working somehow. If I remove the BrowserView from the window, the entire scrollbar works, but with a BrowserView attached, mouse events get ignored for the first few pixels above the top of the view.

I'm not really sure yet why this is happening; hopefully there's a workaround, but I don't see one right now.

**Comment by dbfreem** [comment]
@PalmerAL I checked this on Windows and it doesn't seem to be an issue there. So this must be something with linux.

**Comment by IanDLacy** [comment]
This is happening for me on MacOS too.

Also,

> @PalmerAL I know about the scrolling feature (actually I disabled closing tabs with scroll). I will open a separate issue for the shift+scroll behaviour.

I want to disable this so bad. How did you do it?
